.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,3 +1,12 @@
+# Info
+
```

Comment:
Is this expected to be different from `Description` below? Since the content is no longer rendered, might be confusing to have two related headers.
```suggestion
```